quickwit-index-config/src/config.rs

```diff
@@ -28,6 +28,16 @@ use tantivy::{Document, Order};
 
 use crate::{DocParsingError, QueryParserError, TAGS_FIELD_NAME};
 
+/// Separator used to format tags into `{field_name}:{value}`
+pub const TAG_FIELD_VALUE_SEPARATOR: &str = ":";
+
+/// Wilcard value use to collapse too many tag values into one.
+pub const MANY_TAG_VALUES: &str = "*";
+
+/// Character use to escape tag value when there is collision with the wilcard
+/// tag values.
+pub const TAGS_VALUE_ESCAPE: &str = "\\";
```

Comment:
I'm not sure to understand why we need two backslashes.

Reply:
I am thinking of switching to `char`. `\` alone in a string need to be escaped by itself

Reply:
ah yes indeed. Maybe a char would be nice